src/options.jl

```diff
@@ -0,0 +1,135 @@
+using TOML
+options = Dict{Symbol,Any}(
+    :bdaily_holidays_map => nothing,
+    :bdaily_skip_nans => false, 
+    :bdaily_skip_holidays => false
+)
+
+"""
+    set_option(option::Symbol, value)
+
+Sets the provided option to the provided values.
+
+Current available options are:
+* `:bdaily_holidays_map`: This option holds a tseries of boolean values spanning from 1970-01-01 to 2049-12-31. Values on dates
+    with a `false` entry will not be returned when calling the `values` function on a BDaily TSeries 
+    with the holidays=true option.
+* `:bdaily_skip_nans`: This option controls the treatment of NaN values in BDaily arrays when performing
+    `shift`, `lag`,`diff`, and `pct` functions on them. When true, NaNs are replaced with the most relevant non-NaN value when available.
+* `:bdaily_skip_holidays`: When true, the values function will always be called with holidays=true for BDaily series. 
+    This also controls the behavior of the `shift`, `lag`, `diff`, and `pct` functions, but only NaNs falling on holidays are replaced.
+    
+"""
+function set_option(option::Symbol, value)
+    options[option] = value;
+    nothing #don't return anything
+end
+
+"""
+    get_option(option::Symbol)
+
+Returns the current value of the provided option.
+
+See also [`set_option`](@ref)
+"""
+function get_option(option::Symbol)
+    options[option];
+end
+
+
+"""
+    get_holidays_options(country::Union{String,Nothing}=nothing)
+
+Returns a dictionary of country codes for supported countries and their subdivisions (where applicable)
+
+Holiday calendars are produced using the [python-holidays](https://github.com/dr-prodigy/python-holidays) libary. See their site for more.
+"""
+function get_holidays_options(country::Union{String,Nothing}=nothing)
+    countries = TOML.parsefile(joinpath(replace(@__DIR__, "/src" => ""), "data/holidays.toml"))
```

Comment:
In `{set,get}_holidays_map`: "/src" might not work on Windows. I suggest `joinpath(@__DIR__, "..", "data", "holidays.toml")`, just to be safe.